### PR TITLE
39 implement service view

### PR DESCRIPTION
**Pull request by ghost**
Changed service view for a selected service by switching it to angular material card style. Added edit button for further pop up dialog implementation.

### CODE REVIEW

File: moninag/static/src/ts/app/services/service-detail.component.html [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+            <button (click)="delete()">Delete</button>
+            <button *ngIf="service.state == true" (click)="deactivate()">Deactivate</button>
+            <button *ngIf="service.state == false" (click)="activate()">Activate</button>
+            <div *ngIf="selectedService">
+                <serviceupdate-app [service]='service'></serviceupdate-app>
+            </div>
+        </md-card-actions>
+    </md-card>
+    <checks-list [service]='service'></checks-list>
+</div>
```
Comment (nkoblai) [inline moninag/static/src/ts/app/services/service-detail.component.html]: Missing newline at end of file.

File: moninag/static/src/ts/app/services/service-update.component.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    save(): void {
+        this.servicesService.update(this.service)
+            .subscribe(() => this.goBack());
+    }
+
+    goBack(): void {
+        this.location.back();
+    }
+}
```
Comment (nkoblai) [inline moninag/static/src/ts/app/services/service-update.component.ts:36]: Again.

File: moninag/static/src/ts/app/services/services.service.ts [hunk 3]
```diff
@@ -40,11 +40,31 @@ export class ServicesService {
         let updated_service = {
             name: service.name,
             status: service.status,
+            state: service.state,
         }
         return this.http.put(`${this.servicesUrl}/${service['id']}/`, JSON.stringify(updated_service), options)
             .catch((error: any) => Observable.throw(error.json().error || 'Server error'));
     }
 
+     deactivate(service: Service): Observable<Service[]> {
```
Comment (nkoblai) [inline moninag/static/src/ts/app/services/services.service.ts]: Incorrect tab width.

File: moninag/static/src/ts/app/services/service-update.component.ts (hunk 2 above)
Comment (ghost) [reply moninag/static/src/ts/app/services/service-update.component.ts:36]: fixed! :man_technologist:

**Review by lhalam — APPROVED** [review]

**Pull request merged**